Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp

```diff
@@ -43,9 +43,6 @@ ControllerInterface g_controller_interface;
 //
 void ControllerInterface::Initialize(void* const hwnd)
 {
-	if (m_is_init)
-		return;
```

Comment:
Are you sure multiple initializations won't cause memory leaks? If I recall, initializing twice causes this for some input backends. Granted, that may have been fixed as well.